src/app/features/switch-account-drawer/components/switch-account-list-item.tsx

```diff
@@ -62,35 +58,32 @@ export const SwitchAccountListItem = memo(
     };
 
     const onClickBtcCopyIcon = (e: React.MouseEvent) => {
-      if (!bitcoinAccount) return;
+      if (!bitcoinAddress) return;
       e.stopPropagation();
       setIsShowingSwitchAccountsState(false);
       navigate(RouteUrls.ReceiveBtc, {
-        state: { btcAddress: bitcoinAccount.address, accountIndex: bitcoinAccount.index },
+        state: { btcAddress: bitcoinAddress, accountIndex: currentAccountIndex },
       });
     };
 
     return (
       <AccountListItemLayout
         index={index}
-        stxAddress={stacksAccount?.address}
-        btcAddress={bitcoinAccount?.address}
+        stxAddress={stacksAddress}
+        btcAddress={bitcoinAddress}
```

Comment:
Do these just fill parts of the UI?

What if these are just sections and we had conditional stuff wrapped in components, avoiding hook condiontality

```ts
// where this component returns null if there is no account
<CurrentBtcAccount>{btcAccount => <Box>all the UI/copy functionality</Box>}</CurrentBtcAccount>
```

Reply:
here we provide addresses for all accounts in the list, so we cannot use such approach here?